packages/e2e/src/App.tsx

```diff
@@ -36,8 +37,8 @@ const App = () => {
         onStartPayment={function (): void {
           console.log('onStartPayment');
         }}
-        supportedCurrencies={['USDC']}
-        totalAmount={0.01}
+        supportedCurrencies={['USDC', 'SOL']}
+        totalAmount={0.01} // @TODO bug when also has normalizedPrice
```

Comment:
do we have a ticket for tracking this ?

Reply:
no. let's push this, then go over TODO's and track log them into Linear